CucumberTest/pom.xml

```diff
@@ -0,0 +1,55 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>CucumberTest</groupId>
+  <artifactId>CucumberTest</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <name>CucumberTest</name>
+  <description>CucumberTest</description>
+  

```

Comment:
1. use properties tag maintain single version throughout project
2. explicitly mention java and maven version in pom
3. use final tag for exporting artifact